workers/unity/Packages/com.improbable.gdk.core/Tests/Editmode/GameObjectRepresentation/Readers/ReaderUpdateTests.cs

```diff
@@ -0,0 +1,269 @@
+﻿using System;
```

Comment:
general note: there are a lot of empty lines in methods. maybe remove some of them and create blocks of code to make it easier to read and separate which code belongs together

Reply:
Reconsidering my empty lines

Reply:
Some empty lines were terminated

Reply:
🔫 🤖